types/is-boolean-object/is-boolean-object-tests.ts

```diff
@@ -0,0 +1,15 @@
+import isBoolean = require('is-boolean-object');
+
+isBoolean(Boolean(42)); // $Expects true
+isBoolean(false); // $Expects true
+isBoolean(true); // $Expects true
+
+isBoolean(undefined); // $Expects false
+isBoolean(null); // $Expects false
+isBoolean([]); // $Expects false
+isBoolean({}); // $Expects false
+isBoolean(/a/g); // $Expects false
+isBoolean(new Date()); // $Expects false
+isBoolean(42); // $Expects false
+isBoolean(NaN); // $Expects false
+isBoolean(Infinity); // $Expects false
```

Comment:
```suggestion
isBoolean(Boolean(42)); // $ExpectType boolean
isBoolean(false); // $ExpectType boolean
isBoolean(true); // $ExpectType boolean

isBoolean(undefined); // $ExpectType boolean
isBoolean(null); // $ExpectType boolean
isBoolean([]); // $ExpectType boolean
isBoolean({}); // $ExpectType boolean
isBoolean(/a/g); // $ExpectType boolean
isBoolean(new Date()); // $ExpectType boolean
isBoolean(42); // $ExpectType boolean
isBoolean(NaN); // $ExpectType boolean
isBoolean(Infinity); // $ExpectType boolean
```
this will test outcome of your declaration file (compiler + linter), there is no runtime execution n this repo